### DISCUSSION TITLE
How many dimes and quarters are there? 💰🪙

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1243] [Galaxy Trivia]
There are 20 dimes and quarters in a bag. If the domes were quarters and the quarters were dimes, they would be worth 90 cents more than they are now.

**Comment by Camarota-234** [comment]
There are 13 dimes and 7 quarters in the bag, which results in 305 cents ( 13 x  10 + 7 x 25 = 305).
If the dimes were quarters and the quarters were dimes there would be 7 dimes and 13 quarters in the bag, resulting in 395 cents ( 7 x  10 + 13 x 25 = 395).

**Reply by ParanoidUser** [reply]
There is a typo in the last equation "( 7 x 10 + 13 x 25 = 3**9**5)".

**Comment by KR-Ravindra** [comment]
There are 12 dimes and 8 quarters in the bag.

**Comment by peterbaccaro** [comment]
In order for there to be 90 more cents after the dimes and quarters were substitutes there would need to be 13 dimes and 7 quarters in the bag initially.

- 13 dimes and 7 quarters = 305 cents
- 13 quarters 7 dimes = 395 cents
- 395 cents minus 305 cents = 90 cents

![image](https://github.com/ParanoidUser/yolo/assets/74666079/69b178d2-591a-42b6-9c79-5c27d99abe1b)

**Reply by ParanoidUser** [reply]
I like the approach you have taken here. 👍
